lib/cosmos/script/telemetry.rb

```diff
@@ -164,37 +163,27 @@ def get_tlm_details(items)
 
     # Returns the buffer from the telemetry packet.
     def get_tlm_buffer(target_name, packet_name)
-      return $cmd_tlm_server.get_tlm_buffer(target_name, packet_name)
+      $cmd_tlm_server.get_tlm_buffer(target_name, packet_name)
     end
 
     # Subscribe to one or more telemetry packets. The queue ID is returned for
     # use in get_packet_data and unsubscribe_packet_data.
     # Usage:
     #   id = subscribe_packet_data([[target_name,packet_name], ...], <queue_size>)
     def subscribe_packet_data(packets, queue_size = CmdTlmServer::DEFAULT_PACKET_DATA_QUEUE_SIZE)
-      result = $cmd_tlm_server.subscribe_packet_data(packets, queue_size)
-      result
+      $cmd_tlm_server.subscribe_packet_data(packets, queue_size)
     end
 
     # Unsubscribe to telemetry packets. Pass the queue ID which was returned by
     # the subscribe_packet_data method.
     # Usage:
     #   unsubscribe_packet_data(id)
     def unsubscribe_packet_data(id)
-      result = $cmd_tlm_server.unsubscribe_packet_data(id)
-      result
+      $cmd_tlm_server.unsubscribe_packet_data(id)
     end
 
-    # DEPRECATED
     def get_packet_data(id, non_block = false)
-      results = $cmd_tlm_server.get_packet_data(id, non_block)
-      if Array === results and results[3] and results[4]
-        results[3] = Time.at(results[3], results[4]).sys
-        results.delete_at(4)
-      end
-      results
+      $cmd_tlm_server.get_packet_data(id, non_block)
```

Comment:
get_packet_data is a nice api for systems that don't know the whole definition of everything (like python).  It's only really deprecated from Ruby... so we can't remove it.  

Time doesn't serialize to JSON which is the only reason a time object wasn't sent directly.   Ideally the api should give a ruby time object.   

Having the return value be different between the internal API and the script api is confusing now though...    

Any breaking changes bump us to COSMOS 4.1 which I don't want to goto now.   Need to just keep it around for now and maybe we can remove the Ruby script version in 4.1.